server/api/v1/screenshots/[id]/index.get.ts

```diff
@@ -0,0 +1,17 @@
+// get a specific screenshot
+import aclManager from "~/server/internal/acls";
+import screenshotManager from "~/server/internal/screenshots";
+
+export default defineEventHandler(async (h3) => {
+  const userId = await aclManager.getUserIdACL(h3, ["screenshots:read"]);
+  if (!userId) throw createError({ statusCode: 403 });
+
+  const screenshotId = getRouterParam(h3, "id");
+  if (!screenshotId)
+    throw createError({
+      statusCode: 400,
+      statusMessage: "Missing screenshot ID",
+    });
+
+  return await screenshotManager.get(screenshotId);
```

Comment:
Any verification it's the user's screenshot?

Reply:
totally forgot about that here